README.md

```diff
@@ -0,0 +1,44 @@
+ITKDoxygen
```

Comment:
Should the exiting `readme` file (without extension) be removed?

Reply:
Done. My bad. Thanks Dženan.